src/locales/en.yaml

```diff
@@ -82,6 +82,8 @@ app:
       file_name: Filename
       transfer_rate: Transfer rate
       uploaded: Uploaded
+      open_klipper_config: Open Klipper config
+      open_moonraker_config: Open Moonraker config
```

Comment:
Maybe these can be merged together and replaced with `Open Documentation`? Or maybe something dynamic, like `View temperature_sensor documentation`

Reply:
Done!! I've updated the screenshots with the changes!